docs/development/pluginhooks.md

```diff
@@ -447,13 +456,16 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 
 - Description:
 - Invoked by: `dokku deploy`
-- Arguments: `$APP`
+- Arguments: `$APP $TAG`
 - Example:
 
 ```shell
 #!/usr/bin/env bash
 
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_PATH/common/functions"
+APP="$1"; TAG=$(get_running_image_tag $APP)
+verify_app_name "$APP" "$TAG"
```

Comment:
Yeah I'd rather it be clear as to how variables are set than placing magic in random places like this ;)